package.json

```diff
@@ -78,7 +81,7 @@
     "download": "node ./bin/download-yarn",
     "prepare": "npm run clean && npm run build && npm run download",
     "release": "np",
-    "test": "npm run download && jest -i && flow && standard",
+    "test": "npm run download && jest && flow && standard",
```

Comment:
@dickeyxxx I have not run into any issues with this because it is installing to different directories, though there may be some issues with the yarn cache apparently.  I could add `--mutex` to the yarn options to isolate this at the yarn level.  Thoughts?